src/reactapp/src/components/billingAddress/components/BillingAddressForm.jsx

```diff
@@ -91,31 +91,21 @@ function BillingAddressForm() {
           required
           onKeyDown={handleKeyDown}
         />
-        {selectedCountry &&
-          (hasStateOptions ? (
-            <SelectInput
-              label={__('State')}
-              name={fields.region}
-              required
-              options={stateOptions}
-              onKeyDown={handleKeyDown}
-            />
-          ) : (
-            <TextInput
-              label={__('State')}
-              name={fields.region}
-              placeholder={__('State')}
-              required
-              onKeyDown={handleKeyDown}
-            />
-          ))}
         <SelectInput
           label={__('Country')}
           name={fields.country}
           required
           options={countryOptions}
           onKeyDown={handleKeyDown}
         />
+        <SelectInput
```

Comment:
Why we put region field after country field? is this the order incorrect?

Reply:
This makes sense: You need to select a country first, then the region

Reply:
I just checked this in luma checkout. i finally understood i am wrong. thanks Fab :)